### ISSUE TITLE
Error on GCconnex when searching for Members

### ISSUE THREAD

**Issue by massan35** [gctools-outilsgc/gcconnex#2654]
User is searching for members and colleague using the search Members tool in GCconnex

##Gets this screen when searching by name

![image](https://github.com/gctools-outilsgc/gcconnex/assets/138692330/365c6e06-024b-42e6-8374-e2566be1e76e)

When clicks send anyway, gets this error message

![image](https://github.com/gctools-outilsgc/gcconnex/assets/138692330/a8c3b846-a6ae-46f7-8a35-a37eaee8a04f)

**Comment by Phanoix** [comment]
yes, if anyone is waiting for this to be fixed right now they should just use that and filter to only user results, that's exactly the same page that this would get you.
